target/scripts/helpers/postfix.sh

```diff
@@ -43,21 +43,26 @@ function _vhost_collect_postfix_domains() {
   local DATABASE_VIRTUAL='/tmp/docker-mailserver/postfix-virtual.cf'
   local DOMAIN UNAME
 
-  # getting domains FROM mail accounts
+  # Extract domains from mail accounts:
   if [[ -f ${DATABASE_ACCOUNTS} ]]; then
-    while IFS=$'|' read -r LOGIN _; do
-      DOMAIN=$(echo "${LOGIN}" | cut -d @ -f2)
+    while IFS=$'|' read -r FIRST_FIELD _; do
```

Comment:
While `FIRST_FIELD` is absolutely correct, IMO it'd be more useful to state what the first field actually is and use this as a name.

Reply:
> While `FIRST_FIELD` is absolutely correct, IMO it'd be more useful to state what the first field actually is and use this as a name.

I think it's clear that we're taking the first field and don't care too much about being more descriptive about that as we clearly process that field directly after into the variable(s) `UNAME` / `DOMAIN`?

Before it was `LOGIN` which isn't that much more meaningful. There is a comment directly above the file name check (for the `postfix-accounts.cf` file, that states it's extracting domains from mail accounts / addresses. We know that the expectation is that the first field has a domain-part to extract.

Reply:
According to the comment, would `MAIL_ACCOUNT` make sense?